lab-andrew/entry.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+require('!!file?name=[name].[ext]!./html/index.html');
+require('./scss/base.scss');
+
+const angular = require('angular');
+const cowsay = require('cowsay-browser');
+var app = angular.module('cowsayApp', []);
+
+app.controller('CowsayController', ['$scope', function($scope){
+  $scope.cowsayText = 'And so it begins';
+  $scope.cowsayFile = 'ghostbusters';
+  $scope.prev = [];
+  $scope.count = 0;
+  $scope.say = function() {
```

Comment:
Don't repeat yourself.
You can just "return $scope.result = cowsay.say( //code );"